cluster/node-pools/master-default/stack.yaml

```diff
@@ -37,6 +37,9 @@ Resources:
       - Key: node.kubernetes.io/role
         PropagateAtLaunch: true
         Value: master
+      - Key: zalando.org/pod_max_pids
```

Comment:
`pod-max-pids` maybe? We don't use underscores in other labels.

Reply:
true, good one, will change!